cmd/boulder-va/gsb.go

```diff
@@ -119,6 +120,10 @@ func newGoogleSafeBrowsingV4(gsb *cmd.GoogleSafeBrowsingConfig, logger blog.Logg
 	if err != nil {
 		return nil, err
 	}
+	err = sb.WaitUntilReady(context.Background())
```

Comment:
> Note this will block the start up of boulder-va indefinitely if for some reason the database never becomes healthy.

Is it worth constructing a `context.WithTimeout` here setting some upper bound just for safety's sake? Maybe 60s?